assets/cubyz/biomes/cave/crystal.json

```diff
@@ -2,7 +2,7 @@
 	"isCave" : true,
 	"maxHeight" : -512,
 
-	"chance" : 0.2,
+	"chance" : 1,
```

Comment:
I intentionally made these biomes rarer to make these feel more special when the player discovers them underground.
If you disagree with this decision then please share your reasons for changing it.